lib/ansible/modules/windows/win_package.ps1

```diff
@@ -1268,6 +1268,7 @@ if ($productid -eq $null)
     $productid = Get-Attr -obj $params -name product_id -failifempty $true -resultobj $result
 }
 $arguments = Get-Attr -obj $params -name arguments
+$normalizeargs = Get-Attr -obj $params -name normalize_arguments -type "bool" 
```

Comment:
Could we move the *win_package* module over to the new `Get-AnsibleParam` call and options ?
I know it is an alias, but all our modules should be exemplary to contributors IMO.